tests/functional/spec/introduction.spec.js

```diff
@@ -0,0 +1,20 @@
+const helpers = require('../helpers');
+
+const IntroductionPage = require('../pages/landing.page');
+
+describe('@watch Introduction page', function() {
+
+  const introduction_schema = 'test_introduction.json';
+
+  it('Given I start a survey, When I view the introduction page then I should be able to see introduction information', function() {
+    return helpers.openQuestionnaire(introduction_schema).then(() => {
+      return browser
+        .getUrl().should.eventually.contain(IntroductionPage.pageName)
+        .click(IntroductionPage.showAllButton())
+        .isVisible(IntroductionPage.useOfData().should.eventually.be.true)
```

Comment:
Close brackets are in the wrong place.
Should be .isVisible(IntroductionPage.useOfData()`)`.should.eventually.be.true